docs/mkdocs/docs/technical/releasing.md

```diff
@@ -11,8 +11,8 @@ Click `Run Workflow` on the right hand side:
 1. Type in the new version number
 2. Click `Run workflow`.
 
-Leave `Bump branch to the next version` as `No`. 
-This will create a branch off of `master` incrementing the version in `setup.cfg` but, 
+Leave `Bump branch to the next version` as `No`.
+This will create a branch off of `master` incrementing the version in `setup.cfg` but,
 as of the time of writing, we are leaving that unchanged.
```

Comment:
don't we need to update these notes to describe doing the README update?

Reply:
updated